melodic/distribution.yaml

```diff
@@ -11259,6 +11259,28 @@ repositories:
       url: https://github.com/ros-geographic-info/unique_identifier.git
       version: master
     status: maintained
+  uos_tools:
+    doc:
+      type: git
+      url: https://github.com/uos/uos_tools.git
+      version: master
+    release:
+      packages:
+      - uos_common_urdf
+      - uos_diffdrive_teleop
+      - uos_freespace
+      - uos_gazebo_worlds
+      - uos_maps
+      - uos_tools
+      tags:
+        release: release/melodic/{package}/{version}
+      url: https://github.com/uos-gbp/uos-tools.git
+      version: 1.0.0-1
+    source:
+      type: git
+      url: https://github.com/uos/uos_tools.git
```

Comment:
The package.xml files inside of this repository have `license` tags (good!), but there is no LICENSE file at the top level of the repository (or in the individual package subdirectories).  Can you please add one?  Otherwise this looks good to me.

Reply:
I added a license file to https://github.com/uos/uos_tools

Reply:
Thanks for iterating.